src/main/java/net/snowflake/client/jdbc/SnowflakeDriver.java

```diff
@@ -61,35 +59,6 @@ private static void initializeArrowSupport() {
       disableArrowResultFormat = true;
       disableArrowResultFormatMessage = t.getLocalizedMessage();
     }
-    disableIllegalReflectiveAccessWarning();
-  }
-
-  static void disableIllegalReflectiveAccessWarning() {
```

Comment:
Could you please check why this code was there at first place? is it used by StoredProc team?

Reply:
> Could you please check why this code was there at first place? is it used by StoredProc team?

Hey @sfc-gh-igarish this was the original PR: https://github.com/snowflakedb/snowflake-jdbc/pull/377. I can't find a strong reason to suppress the logger as it will affect the entire application.

Reply:
@sfc-gh-igarish Can we proceed to merge this PR?

Reply:
Just to provide some context, the original reason for adding PR https://github.com/snowflakedb/snowflake-jdbc/pull/377 was that some customers requested us to suppress the illegal reflective access warnings, as that breaks their applications.

Reply:
@sfc-gh-wshangguan Can you share the original issue, I can't find it in the PR